File: src/integrationTest/resources/projects/javagradleplugin/src/main/java/GradlePlugin.java
// SPDX-FileCopyrightText: 2022 Sveriges Television AB
//
// SPDX-License-Identifier: Apache-2.0

import org.gradle.api.Plugin;
import org.gradle.api.Project;

public class GradlePlugin implements Plugin<Project> {
    public void apply(Project project) {
        project.getTasks().register("greeting", task -> {
            task.doLast(s -> System.out.println("Hello from AppPlugin"));
        });
    }
}
    /*plugins {
    `java-gradle-plugin`
    }*/

File: src/integrationTest/resources/projects/kotlingradleplugin/src/main/kotlin/GradlePlugin.kt
// SPDX-FileCopyrightText: 2021 Sveriges Television AB
//
// SPDX-License-Identifier: Apache-2.0

import org.gradle.api.Plugin
import org.gradle.api.Project

class GradlePlugin : Plugin<Project> {
    override fun apply(project: Project) {
        project.tasks.register("greeting") { task ->
            task.doLast {
                println("Hello from plugin 'hejsan.greeting'")
            }
        }
    }
}
/*plugins {
    id("org.jetbrains.kotlin.jvm") version "1.5.0"

    // Apply the java-library plugin for API and implementation separation.
    `java-library`
}*/
